app/src/main/java/org/oppia/android/app/policies/PoliciesFragmentPresenter.kt

```diff
@@ -78,6 +80,9 @@ class PoliciesFragmentPresenter @Inject constructor(
       supportsLinks = true,
       supportsConceptCards = false
     )
+
+    val websitePattern = Pattern.compile("https://www.oppia.org")
```

Comment:
This pattern will only be useful for finding links to this address, but if we would like to find **any** link in the span, we may use something like:
```suggestion
    val websitePattern = Pattern.compile("(http|ftp|https)://([\\w_-]+(?:(?:\\.[\\w_-]+)+))([\\w.,@?^=%&:/~+#-]*[\\w@?^=%&/~+#-])")
```

Reply:
cool, let me test this and see if it works, thanks @adhiamboperes